core/src/main/java/org/htsjdk/core/api/PathURI.java

```diff
@@ -0,0 +1,86 @@
+package org.htsjdk.core.api;
```

Comment:
I would say that a better place will be `org.htsjdk.core.api.io`, to keep things more organized.

Reply:
Done.